analyzers/tests/SonarAnalyzer.UnitTest/Helpers/MethodSignatureHelperTest.cs

```diff
@@ -124,7 +124,7 @@ End Namespace
         }
 
         [TestMethod]
-        public void IsMatch_AndCheckingOverrides_DoesMatchOverrides_CS()
+        public void MatchesAny_AndCheckingOverrides_DoesMatchOverrides_CS()
```

Comment:
- I think we can rename the test file to `MemberDescriptorTest` since it appears to test only the `MemberDescriptor`.
 - I've checked and none of the tests from `MethodSignatureHelperTest.cs` calls `IsMatch`. It seems to be covered only indirectly by other tests.

Reply:
(I'm not sure the names are perfect now either, but I think they're ok-ish)